pages/linux/last.md

```diff
@@ -0,0 +1,37 @@
+# last
+
+> List information of user logins.
+> See also: `lastb`, `tac`.
+> More information: <https://manned.org/last.1>.
+
+- List login information (e.g., username, terminal, boot time, kernel):
+
+`last`
+
+- List login information of a specific user:
+
+`last {{username}}`
+
+- List information of a specific TTY:
+
+`last {{tty1}}`
+
+- List most recent information (by default, the newest are at the top):
+
+`last | tac`
+
+- List information of system boots:
+
+`last "{{system boot}}"`
```

Comment:
I did a search in the repo and it seems like this is the preferred form (`"{{arg}}"` instead of `{{"arg"}}`), but there are still a few cases of `{{"arg"}}`. Is this style preference documented anywhere?

Reply:
> I did a search in the repo and it seems like this is the preferred form (`"{{arg}}"` instead of `{{"arg"}}`), but there are still a few cases of `{{"arg"}}`. Is this style preference documented anywhere?

I don't think we have documented it yet, but yes we do prefer `"{{placeholder}}"`.

Reply:
For me, `"{{argument}}"` is better because it emphasizes the placeholder itself a bit more also just suggests the user to change the contents of the argument itself.

Reply:
> In which scenario would quotes around of the curly braces be preferred?

It is preferred for placeholders supporting strings as arguments.